R/Widget_FlagOverTime.R

```diff
@@ -24,30 +24,51 @@ Widget_FlagOverTime <- function(
   dfResults,
   dfMetrics,
   strGroupLevel = c("Site", "Study", "Country"),
-  strFootnote = NULL
+  strFootnote = NULL,
+  width = "100%",
+  height = "400px"
 ) {
   stopifnot(
     "dfResults is not a data.frame" = is.data.frame(dfResults),
     "dfMetrics is not a data.frame" = is.data.frame(dfMetrics),
     "strGroupLevel is not a character" = is.character(strGroupLevel)
   )
 
+  most_recent12 <- dfResults %>%
+    pull(SnapshotDate) %>%
+    unique() %>%
+    sort(decreasing = TRUE) %>%
+    head(12) %>%
```

Comment:
Curious what if snapshots are collected biweekly, quarterly, different frequencies, is there a more "flexible" way to be robust for the user?

Reply:
in general, we collect monthly snapshots, but even if it were collected weekly, this would provide roughly a quarter-long view, so i think its a reasonable duration to show as the default